install_config/cluster_metrics.adoc

```diff
@@ -544,3 +544,19 @@ For more information about Heapster and how to access its APIs, please refer the
 link:https://github.com/kubernetes/heapster/[Heapster] project.
 
 endif::[]
+
+== Cleanup
```

Comment:
Can you add the following unique anchor on the line before the section title (we have another use of "cleanup" elsewhere in the guides):

```
[[metrics-cleanup]]
```